docs/advanced-usage/pytorch-classifier.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 4
+id: pytorch-classifier
+title: Integrating Your Custom PyTorch Image Classifier
+---
+
+In this guide, we will walk you through the process of leveraging the `TORCHSCRIPT_CLASSIFIER` node in Flojoy to integrate a custom image classification model. Specifically, we'll use a Colab notebook to train the `MobileNetV3` model on a cats and dogs dataset, export it as a `.torchscript` format, and then deploy it within the `TORCHSCRIPT_CLASSIFIER` node.
+
+### 1. Train and Export Your Model
+
+Start by accessing the provided Colab notebook which serves as an ideal foundation for training and exporting your model. This notebook is tailored for a cats vs. dogs dataset, but you can effortlessly modify it to suit different datasets with varying class numbers.
+
+- **Colab Notebook**: [Link](https://colab.research.google.com/drive/1HCj3i43tYt4CTLPHq4BqVC2HL0lAN2-X?usp=sharing)
```

Comment:
Side-note: this is convenient, but this constraints Flojoy's documentation to a third-party service (Google Colab).

Reply:
Hmm great point, although Colab is a third-party service, I am confident the benefits outweigh the disadvantages, and here are a few reasons why

- Colab is here to stay in the ML ecosystem and remain freely accessible
- It provides a great reference implementation for people to compare with their own in-house implementation
- Allows anyone to quickly and instantly get started with a premade Cats and Dogs example
- Is more readable and interactive than a Python script
- Will feel very familiar to people who have used it in the past

Reply:
P.S: I added a disclaimer at the beginning of the tutorial to help drive the point home

Reply:
Thanks!

I also think Colab is adapted and using it here is relevant for all the points you indicated. People might not have access to GPU and Colab is handy to access some.

I was unclear, what I meant was that we could indicate it up-front and your disclaimer does it.